src/cli.ts

```diff
@@ -87,6 +90,21 @@ async function main() {
     };
   }
 
+  function parseGoLangOptions() {
+    const outdir = argv['golang-outdir'];
+    const module = argv['golang-module'];
+    if (!outdir && !module) { return undefined; }
+    if (!outdir) { throw new Error('--golang-outdir is required'); }
+    if (!module) { throw new Error('--golang-module is required'); }
+    return {
+      // moduleKey: module,
```

Comment:
This is where you could explicitly set `moduleKey` for `golang` but then it breaks other languages

Reply:
Hey @campionfellin, could you elaborate on how it does break the other languages. I tried it locally and it seemed to still work. I'd think it shouldn't matter to JSII whether there's a random hash or some actual name if the result does not contain that hash/name anyway? Did I miss something?

Reply:
I can't quite remember here... I think it had something to do with `moduleKey` needing to be a random hash for some languages, but needed to be named for `go`. Perhaps I just couldn't figure it out at the time, but now it sounds like it's working for you!